indicators/serializers_new/participant_count_serializers.py

```diff
@@ -297,9 +297,9 @@ def disaggregations_are_valid(self, used_disaggregations):
             # Translators: An error message detailing that the sum of 'SADD without double counting' should be equal to the sum of 'Direct without double counting'
             raise exceptions.ValidationError(_("The sum of 'SADD without double counting' should be equal to the sum of 'Direct without double counting'."))
 
-        if actual_with_direct == 0 or actual_with_indirect == 0:
-            # Translators: An error message detailing that the fields Direct and Indirect total participants with double counting is required
-            raise exceptions.ValidationError(_("Direct/indirect total participants with double counting is required. Please complete these fields."))
+        if actual_with_direct is None or actual_with_indirect is None:
```

Comment:
Does this still raise an error if indirect is left empty or 0?

Reply:
It shouldn't raise an error. Empty values are handled before this method is called, and 0 values shouldn't raise the error. I tested with the frontend changes and could enter 0 or empty values.